pallas-crypto/src/hash/hasher.rs

```diff
@@ -78,6 +78,17 @@ macro_rules! common_hasher {
                 hasher.finalize()
             }
 
+            #[inline]
+            pub fn hash_tagged_cbor(
+                data: &impl minicbor::Encode,
+                tag: &[u8],
```

Comment:
We could also use a single `u8` if preferred as this will likely cover all use cases for a long time.